Base.gs

```diff
@@ -1,29 +1,84 @@
-//an example for some basic Google Apps Script / Bot stuff
+// An example for some basic Google Apps Script / Bot stuff
 
 var botId = "your bot id here";
-function sendText(text){
+var accessToken = "your access token here";
+function sendText(text) {
   UrlFetchApp.fetch("https://api.groupme.com/v3/bots/post", {"method":"post", "payload":'{"bot_id":"' + botId + '","text":"' + text + '"}'})
 }
 
-function sendImage(text, imageURL){
+function sendImage(text, imageURL) {
   UrlFetchApp.fetch("https://api.groupme.com/v3/bots/post", {"method":"post", "payload":'{"bot_id":"' + botId + '","text":"' + text + '","attachments":[{"type":"image","url":"' + imageURL + '"}]}'})
 }
 
-//respond to messages sent to the group. Recieved as POST 
-function doPost(e){
+// Will include the "message" parameter appended after the "username" string, with the "username" parameter bolded
+// Mentioning a user by using an invalid ID (like 0) allows you to bold text without mentioning an actual user
+function mentionUser(message, username, userId) {
+  UrlFetchApp.fetch("https://api.groupme.com/v3/bots/post", {"method":"post", "payload":'{"bot_id":"' + botId + '","text": "@' + username + ' ' + message + '", ' + 
+                                                               '"attachments": [{"type": "mentions", "loci": [[0,' + 
+                                                             username.length + ']], "user_ids": ["' + userId + '"]}]}'});
```

Comment:
i think some of this stuff might be a bit cleaner to read with a template literal instead of concatenating strings all over. what do you think?

Reply:
Yeah I'll work on creating a quick example (probably an @all bot) and add it to the readme, then can clean up that long string